Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/securitycontextconstraints/selinux/runasany_test.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package selinux
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"testing"
+)
+
+func TestRunAsAnyOptions(t *testing.T) {
+	_, err := NewRunAsAny(nil)
+	if err != nil {
+		t.Fatalf("unexpected error initializing NewRunAsAny %v", err)
+	}
+	_, err = NewRunAsAny(&api.SELinuxContextStrategyOptions{})
+	if err != nil {
+		t.Errorf("unexpected error initializing NewRunAsAny %v", err)
+	}
+}
+
+func TestRunAsAnyGenerate(t *testing.T) {
+	s, err := NewRunAsAny(&api.SELinuxContextStrategyOptions{})
+	if err != nil {
+		t.Fatalf("unexpected error initializing NewRunAsAny %v", err)
+	}
+	uid, err := s.Generate(nil, nil)
+	if uid != nil {
+		t.Errorf("expected nil uid but got %d", *uid)
+	}
+	if err != nil {
+		t.Errorf("unexpected error generating uid %v", err)
+	}
+}
+
+func TestRunAsAnyValidate(t *testing.T) {
+	s, err := NewRunAsAny(&api.SELinuxContextStrategyOptions{})
```

Comment:
You might consider passing a few different options in and show that they all pass validate.